### PR TITLE
feat(controller): Add controller to postcard creation form

### PR DESCRIPTION
**Pull request by shraddha1704**
### What
Add controller and controller-tests to postcard creation form

### Why
Angular bootcamp

### CODE REVIEW

File: app/create/controller.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+module.exports = function ($scope, Postcard) {
+
```
Comment (jesscxu) [inline app/create/controller.js]: nit: remove this line?

File: app/create/controller.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  $scope.payload = { mail_type: 'usps_first_class' };
+  $scope.loading = false;
+  $scope.error = null;
+  $scope.postcard = null;
+
+  $scope.create = function (payload) {
+    $scope.loading = true;
+    $scope.error = null;
```
Comment (jesscxu) [inline app/create/controller.js:11]: do we need this line and the next? idk haha

File: test/create/controller.spec.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  }));
+
+  describe('create', function () {
+
+    beforeEach(function () {
+      $controller('CreateCtrl', {
+        $scope: $scope
+      });
+
+      $scope.$apply();
```
Comment (jesscxu) [inline test/create/controller.spec.js:30]: do you need this apply here?

**Review by jesscxu — COMMENTED** [review]
LGTM

also can we put `.DS_Store` in the `.gitignore`?

File: app/create/controller.js (hunk 2 above)
Comment (shraddha1704) [reply app/create/controller.js:11]: Yes. On every new request, it needs to cleared out.
[Reference notion doc](https://www.notion.so/lob/Controllers-46fa970c00ae481f8bbb1a37d5dfc03c#ce532bd037524fcda86e4cf3430dffa6)

**Review by nyergler — APPROVED** [review]

**Pull request merged**